index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>About Me-JoeG</title>
+</head>
+<h1>About Me</h1>
+
+<body>
+    <img src=https://images.freeimages.com/images/large-previews/3d5/portland-panorama-1228838.jpg> <section>
+    <h1>A blurb about my me.</h1>
+    <p>Here is a list of a few things I love and a few things I hate. Pay attention!! There is a quiz. You will get a
+        prize if you score 90% or higher but you need ask me for it.</p>
+    </section>
+
+    <section>
+        <h2>Things I love!!</h2>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sunt illum rerum quam. Magnam rem sed nemo
+            voluptates
+            amet delectus error. Id, quibusdam inventore. Laboriosam, excepturi vero? Officiis nostrum tenetur voluptas!
+        </p>
+
+        <ol>
+            <li>Stereo's "two channel audio"</li>
+            <li>Cars</li>
+            <li>All Things Tech</li>
+        </ol>
+    </section>
+
+    <section>
+        <h2>Things I hate!!</h2>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sunt illum rerum quam. Magnam rem sed nemo
+            voluptates
+            amet delectus error. Id, quibusdam inventore. Laboriosam, excepturi vero? Officiis nostrum tenetur voluptas!
+        </p>
+
+        <ul>
+            <li>Traffic</li>
+            <li>Key-Lime Pie</li>
+            <li>Polenta</li>
+        </ul>
+    </section>
+
+    <div>
+        <button id="quizbutton">Take My Quiz!</button>
+    </div>
+    <section id="Score">
```

Comment:
remember to use kebab case for html. Line 49 should be `quiz-button`. Line 51 should just be `score` without the capital 's'